lib/markdown2.py

```diff
@@ -362,6 +366,9 @@ def convert(self, text):
         if "fenced-code-blocks" in self.extras and self.safe_mode:
             text = self._do_fenced_code_blocks(text)
 
+        if 'admonitions' in self.extras and self.safe_mode:
```

Comment:
Why ran twice within the convert method?

Reply:
My bad, this is something that I was meant to correct in [333ba1c](https://github.com/trentm/python-markdown2/pull/436/commits/333ba1c207d0bb2cd4bf94c8268a94880a7c5f8c).
This was intentional back when I had admonitions being processed before code blocks but is no longer required.